distributed/rtorrent/DEPENDS

```diff
@@ -1,3 +1,7 @@
 depends  libtorrent
 depends  ncurses
 depends  curl
+depends  cppunit
```

Comment:
I'm pretty sure this PR will build just fine if you move "depends cppunit" to libtorrent/DEPENDS instead :)